observations/README.md

```diff
@@ -0,0 +1,1279 @@
+- CompoundStatement (**[Blood Pressure](#Blood-Pressure-(JSON-FHIR->-XML-HL7))**, **[Test Group Header](#Test-Group-Header-(JSON-FHIR->-XML-HL7))**)
+- NarrativeStatement (**[Comment Note](#Comment-Note-(JSON-FHIR->-XML-HL7))** / **[Filing Comment](#Filing-Comment-(JSON-FHIR->-XML-HL7))**)
+
+## Uncategorised Data (JSON FHIR > XML HL7)
+
+Uncategorised data is mapped from an FHIR `Observation` to a HL7 `ObservationStatement` where it is not deemed to be a 
+Blood Pressure or Comment Note. 
+
+| Mapped to (XML HL7 ObservationStatement)             | Mapped from (JSON FHIR / other source )                                                                                                                                         |
+|------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id \[@root]                                          | Unique ID generated by the adaptor                                                                                                                                              |
+| code                                                 | `Observation.code.coding` as described in the FHIR > XML section of [Codeable Concept](../codeable%20concept/README.md)                                                         |
+| statusCode                                           | fixed value = `"COMPLETE"`                                                                                                                                                      |
+| effectiveTime                                        | `Observation.effectiveDateTime` or else `Observation.effectivePeriod`                                                                                                           |
+| availabilityTime                                     | `Observation.effectiveDateTime` or else `Observation.effectivePeriod.start`                                                                                                     | 
+| value                                                | `Observation.valueQuantity` or `Observation.valueString`                                                                                                                        |
+| pertinentInformation / sequenceNumber \[@value]      | fixed value = `"+1"`                                                                                                                                                            |
+| pertinentInformation / pertinentAnnotation / text    | `Observation.bodySite.text` or else `Observation.bodySite.coding.extension[index].value` where `Observation.bodySite.coding.extension[index].url` equals `"descriptionDisplay"` |
```

Comment:
The `ObservationStatementMapper::prepareComment` method looks different to your explanation, am I confused?

Reply:
I overlooked this before, it turned out to be quite completed, so I've added a new section